docs/AppendixA-Troubleshooting_guideline.md

```diff
@@ -35,11 +35,11 @@ _**Table 3: Common errors encountered during execution of the LORIS-MRi insertio
 |`ERROR: This class is designed around the notion of a 'Study'. You can't use it with data from multiple studies.  The following study UIDs were found: '1.3.12.2.1107.66060.300000004' '1.3.12.2.1107.66060.300000007' The dicomTar execution has failed`|The upload contains acquisitions from two scanning sessions|Separate into two separate uploads|
 |`Out of memory!` during the execution of `dicomTar.pl`|The Transfer syntax of the uploaded scan is other than Little Endian Explicit (such as Implicit or JPEG lossless)|Use the DICOM toolkit `dcmconv` to convert. An example command that changes the transfer syntax to Little Endian Explicit on all DICOM files within a given folder is: `for i in $(find -type f); do dcmconv --write-xfer-little $i $i; done`|
-|`Number of MINC files that will be considered for inserting into the database: 0. No data could be converted into valid MINC files. Localizers will not be considered!`|The upload contains only ‘localizer’ type SeriesDescription|Localizers are not processed by default in LORIS|
-|My uploaded DICOM study contains 6 modalities but only 5 were inserted. The 6th modality was not converted to MINC, neither inserted into the MRI Violated Scans module. In addition, during insertion, the message `Number of MINC files that will be considered for inserting into the database: 5` is displayed, confirming that only 5 files are considered|The missing modality is probably a `localizer` acquisition. `tarchiveLoader` is specifically 'instructed' to exclude this modality|No action needed. This is the expected behavior of the LORIS-MRI insertion pipeline|
+|`Number of MINC files that will be considered for inserting into the database: 0. No data could be converted into valid MINC files.` Your project's excluded series descriptions `will not be considered!`|The upload contains only acquisitions that have `SeriesDescriptions` which are excluded by the project|Projects can specify what acquisitions' `SeriesDescription` are not to be processed by default in LORIS|
+|My uploaded DICOM study contains 6 modalities but only 5 were inserted. The 6th modality was not converted to MINC, neither inserted into the MRI Violated Scans module. In addition, during insertion, the message `Number of MINC files that will be considered for inserting into the database: 5` is displayed, confirming that only 5 files are considered|The missing modality is probably matching one of the excluded series descriptions set in the Config Module under the Imaging Pipeline section. `tarchiveLoader` is specifically 'instructed' to exclude those modalities|Make sure these excluded series  description have correctly been set (note, they will be used for regex  matching)|
```

Comment:
at the end of this sentence
it should be `Make sure these excluded series  descriptionS have correctly` (extra S is missing in the word description; otherwise the verb is not conjugated properly)